except_star.md

```diff
@@ -294,81 +339,82 @@ try:
     "msg", [ValueError('a'), TypeError('b'), TypeError('c'), KeyError('e')]
   )
 except *ValueError as e:
-  print(f'got some ValueErrors: {e}')
+  print(f'got some ValueErrors: {e!r}')
 except *TypeError as e:
-  print(f'got some TypeErrors: {e}')
+  print(f'got some TypeErrors: {e!r}')
   raise
 ```
 
 The above code would print:
 
 ```
-got some ValueErrors: ExceptionGroup("msg", [ValueError('a')])
-got some TypeErrors: ExceptionGroup("msg", TypeError[('b'), TypeError('c')])
+got some ValueErrors: ExceptionGroup('msg', [ValueError('a')])
+got some TypeErrors: ExceptionGroup('msg', [TypeError('b'), TypeError('c')])
 ```
 
-and then terminate with an unhandled `ExceptionGroup`:
+and then terminate raising the following `ExceptionGroup` containing the
+exceptions that were not handled or were caught and reraised:
 
 ```
 ExceptionGroup(
   "msg",
-  [TypeError('b'),
-   TypeError('c'),
-   KeyError('e')]
+  [KeyError('e'),     # unhandled
+   TypeError('b'),    # reraised
+   TypeError('c')]    # reraised
 )
 ```
 
 Basically, before interpreting `except *` clauses, the interpreter will
 have an "incoming" `ExceptionGroup` object with a list of exceptions in it
 to handle, and then:
 
-* The interpreter creates two new empty result lists for the exceptions that
+* The interpreter creates two new empty result lists for any exceptions that
 will be raised in the except* blocks: a "reraised" list for the naked raises
-and a "raised" list of the parameterised raises.
+and a "raised" list of the parameterised raises. It also saves the original
+"incoming" group which it uses in the end, if necessary, to combine the
+reraised exceptions into the same structure.
 
 * Every `except *` clause, run from top to bottom, can match a subset of the
-  exceptions out of the group forming a "working set" of errors for the
-  current clause.  These exceptions are removed from the "incoming" group.
-  If the except block raises an exception, that exception is added to the
+  exceptions out of the "incoming" group, forming a "working set" of errors for
+  the current clause (the value of `sys.exc_info()` while the clause is
+  executed).  These exceptions are removed from the "incoming" group so
+  that they will be excluded from matching additional clauses.
+  If an except block raises an exception, that exception is added to the
   appropriate result list ("raised" or "reraised"), and in the case of "raise"
   it gets its "working set" of errors linked to it via the `__context__`
   attribute.
 
-* After there are no more `except*` clauses to evaluate, there are the
-  following possibilities:
+* After all `except*` clause were evaluated, there are the following
+ possibilities:
```

Comment:
Reading the numbered items carefully, they seem not exclusive -- it's more like there are three steps that are carried out in order? (To me, "possibilities" makes me anticipate three exclusive situations.)